hdfs_datanode/tests/conftest.py

```diff
@@ -0,0 +1,46 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+# stdlib
+import os
+import json
+from mock import patch
+
+# 3rd party
+import pytest
+
+from .common import HERE
+
+
+@pytest.fixture
+def aggregator():
+    from datadog_checks.stubs import aggregator
+    aggregator.reset()
+    return aggregator
+
+
+@pytest.fixture(scope='session')
```

Comment:
In order to avoid having to make this a test function argument, let's make this `@pytest.fixture(scope="session", autouse=True)`.